packages/quantic/force-app/main/default/lwc/quanticHeadlessLoader/quanticHeadlessLoader.js

```diff
@@ -116,6 +116,20 @@ async function initEngine(engineId) {
   }
 }
 
+/**
+ * initialize coveoHeadless Store object 
+ * @param {string} engineId The id of the engine.
+ */
+const initQuanticStore = (engineId) => {
+  try {
+     if(!window.coveoHeadless[engineId].bindings.store) {
+       window.coveoHeadless[engineId].bindings.store = Store.initialize();
+     }
+  } catch (error) {
+    throw new Error('Fatal error: unable to initialize quantic Store: ' + error);
```

Comment:
I think "Quantic" should have a capital letter instead of "store".

```suggestion
    throw new Error('Fatal error: unable to initialize Quantic store: ' + error);
```